app/views/ChooseProvider.js

```diff
@@ -356,7 +386,7 @@ class ChooseProviderScreen extends Component {
                   urlEntryInProgress: true,
                 });
               }}>
-              <Typography style={styles.menuOptionText}>
+              <Typography style={styles.menuOptionText} use={Type.Body2}>
```

Comment:
This wasn't you, but please wrap this lower slide UI into a `SafeAreaView` so that it does not show too low down on the phone:

<img width="342" alt="Screen Shot 2020-04-24 at 4 30 16 PM" src="https://user-images.githubusercontent.com/702990/80264705-d3a8a180-8649-11ea-8c91-3aa5969d2ec6.png">

Reply:
Actually, out of scope. can fix after.